src/modules/settlements/screens/TransactionDetailsScreen.tsx

```
@@ -0,0 +1,285 @@
+          <Text style={styles.errorTitle}>Payment Not Found</Text>
+          <Text style={styles.errorText}>
+            This payment could not be loaded. It may have been deleted.
+          </Text>
+          <Button title="Go Back" onPress={() => router.back()} />
+        </View>
+      </View>
+    );
+  }
+
+  const showCurrencyConversion =
+    settlement.originalCurrency !== settlement.originalCurrency; // TODO: Get trip currency
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)